sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/variant/VariantExpressionEvalUtilsSuite.scala

```diff
@@ -122,4 +122,42 @@ class VariantExpressionEvalUtilsSuite extends SparkFunSuite {
         Map("sizeLimit" -> "16.0 MiB", "functionName" -> "`parse_json`"))
     }
   }
+
+  test("isVariantNull") {
```

Comment:
Recommend to also add SQL tests in `VariantEndToEndSuite`.

Reply:
+1, we can test a few basic cases to prove the integration part works well.

Reply:
I added tests where you can see how it behaves when integrated with other variant expressions, namely `parse_json` and `variant_get`.